charts/agent/README-sub.md.gotmpl

```diff
@@ -1,16 +1,11 @@
 {{ template "chart.header" . }}
```

Comment:
Tested it again and just using `helm-docs` with `README.md.gotmpl` worked quite fine without moving it into server and agent. It will also update the main README, but for that the CI is just running `helm-docs -g . -t README-main.md.gotmpl` afterwards in release.

Reply:
I can't follow, I didn't split into server/agent but into main chart/sub charts. This is how it was before, just more clear now with two dedicated templates. Before, one was used implicitly while the other had an unclear name, i.e. it's scope was not really clear.

Reply:
@anbraten 
I am not sure if we're on the same page here but what I currently see is the following:

- Invoking `helm-docs .` updates all READMEs but doesn't use `README-main.md.gotmpl` for the super chart, leading to wrong changes
- Which is why I would refrain from running this and which is why I added explicit make targets which do the "right" thing, i.e. invoking `helm-docs` with the correct args for both the super chart and the sub-sharts

`make docs` does the right thing now and I've just made another edit in the "Contributing section" telling users to use this command instead of using `helm-docs` directly